doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -467,58 +467,309 @@ The following table indicates the software maturity levels of the support for ea
         - nRF9151
         - nRF9160
         - nRF9161
-      * - **Bluetooth LE Peripheral/Central**
+      * - **2 Mbps PHY**
+        - Supported
+        - Supported
+        - Supported
+        - Supported
+        - Supported
         - Supported
         - Supported
         - Experimental
         - Supported
+        - --
+        - --
+        - --
+        - --
+      * - **Coded PHY (Long Range)**
```

Comment:
for the 52 series, the following devices support Coded:
- 52811
- 52820
- 52833
- 52840